src/com/hms_networks/americas/sc/taginfo/TagInfoManager.java

```diff
@@ -44,18 +48,30 @@
   /** Maximum capacity for byte stream buffer. */
   private static final int MAX_CAPACITY_BYTES = 5000;
 
+  /** Flag to enable int to string tag enums. Defaults to not enabled. */
+  private static boolean enableIntToStringEnums = false;
+
+  /** ArrayList to hold the names of tags that will have int to string enumerations */
+  private static ArrayList intToStringEnumerationTagList;
+
+  /** File path to the int to string enumeration file. */
+  private static final String INT_TO_STRING_ENUMERATION_FILE_NAME = "/usr/EnumerationMapping.json";
+
   /**
    * Populate the tag information list by using an Ewon Export Block Descriptor and parsing the
    * response.
    *
    * @throws IOException if EDB fails
    * @throws TagInfoBufferException if line from var_lst exceeds max capacity
+   * @throws JSONException if string enumeration JSON parse fails
```

Comment:
You add the `@throws JSONException` in many places throughout this class, but the messages are not consistent. Most of them say `if int to string enumeration JSON parse fails` so this one should be updated to that, and any other inconsistent occurrences.